src/jira/client/jira-client.ts

```diff
@@ -33,7 +33,7 @@ export const getJiraClient = async (
 	jiraHost: string,
 	gitHubInstallationId: number,
 	log: Logger = getLogger("jira-client"),
-	gitHubAppId?: number
+	gitHubAppId: number | undefined
```

Comment:
if you're going to do this change, can you please put the log as being the last parameter?  The log isn't important and you should never have a require parameter _after_ an optional one.

Reply:
fixed.